uppsrc/ide/Core/Host.cpp

```diff
@@ -5,8 +5,19 @@
 #define LLOG(x)
 #define METHOD_NAME "Host::" << UPP_FUNCTION_NAME << "(): "
 
-Host::Host()
+#ifdef FLATPAK
+const char* Host::CMDLINE_PREFIX = "host-spawn --no-pty ";
+#else
+const char* Host::CMDLINE_PREFIX = "";
+#endif
+
+String Host::AddCmdlinePrefix(const char* cmdline)
```

Comment:
You seem to be using CMDLINE_PREFIX for the same role as AddCmdlinePrefix, except it does String conversion. If you CMDLINE_PREFIX was String instead of const char *, I think this function could be removed. Less clutter, better code.